game.js

```diff
@@ -4,6 +4,7 @@ const { readFileSync } = require('fs');
 const { Readme } = require('./lib/readme');
 
 const username = process.env["USER"]
+var delayedFunc = function () {}
```

Comment:
Can you break these brackets onto separate lines?  See the legal/illegal moves thing in the readme